modules/op-pipelines-as-code-resolver-annotations.adoc

```diff
@@ -0,0 +1,9 @@
+// This module is included in the following assemblies:
+// * pac/creating-pipeline-runs-pac.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="pipelines-as-code-resolver-annotations_{context}"]
+= {pac} resolver annotations
+
+[role="_abstract"]
+You can use {pac} resolver annotations to reference `Task` and `Pipeline` custom resource (CR) definitions. If you reference a remote task in a pipeline run, or a pipeline in a `PipelineRun` or a `PipelineSpec` object, the {pac} resolver automatically resolves the referenced resource includes it in the resulting `PipelineRun` custom resource (CR). If there is any error while fetching the remote tasks or parsing them, {pac} stops processing the tasks.
```

Comment:
pac actually always fetch the pipeline/tasks from the annotations and use the referenced pipelinens/tasks if they are used as a reference in the pipeline...